src/module.playground/setup/setup.dex.ts

```diff
@@ -121,4 +121,20 @@ export class SetupDex extends PlaygroundSetup<PoolPairSetup> {
       return false
     }
   }
+
+  async before (list: PoolPairSetup[]): Promise<void> {}
+
+  async after (list: PoolPairSetup[]): Promise<void> {
```

Comment:
```suggestion
  protected async after (list: PoolPairSetup[]): Promise<void> {
```

They are meant to be overridable but optional, hence you don't need to declare if you don't need them.